code/game/machinery/cryo.dm

```diff
@@ -637,6 +637,20 @@ var/global/list/cryo_health_indicator = list(	"full" = image("icon" = 'icons/obj
 	put_mob(usr)
 	return
 
+/obj/machinery/atmospherics/unary/cryo_cell/verb/remove_beaker()
+	set name = "Remove beaker"
+	set category = "Object"
+	set src in oview(1)
+	if(usr.incapacitated() || usr.lying || usr.locked_to || !(iscarbon(usr) || issilicon(usr))) //are you cuffed, dying, lying, stunned or other
+		return
+	if(panel_open)
+		to_chat(usr, "<span class='bnotice'>Close the maintenance panel first.</span>")
+		return
+	if(beaker)// If there is, effectively, a beaker
+		detach()
+	add_fingerprint(usr)
+	return
```

Comment:
return at the end of the proc does nothing

Reply:
Whoops